Sources/GoTrue/GoTrueClient.swift

```diff
@@ -23,17 +24,22 @@ public final class GoTrueClient {
       return try await env.sessionManager.session()
     }
   }
+    
+  public let flowType: FlowType!
```

Comment:
No need to be a unwrapped optional.

```suggestion
  public let flowType: FlowType
```